llvm/include/llvm/Transforms/Utils/Instrumentation.h

```diff
@@ -158,6 +158,7 @@ struct SanitizerCoverageOptions {
   bool PCTable = false;
   bool NoPrune = false;
   bool StackDepth = false;
+  int StackDepthCallbackMin = 0;
```

Comment:
Could it make sense to put this at the end of the `bool` list? I wonder if it would make the struct size smaller or not. TBH, this struct should probably use `llvm/include/llvm/ADT/Bitfields.h` for the booleans.

Reply:
Yeah, I wasn't sure if I should keep it with the other StackDepth var. I've moved it now.